contracts/src/v0.8/automation/dev/v2_3/AutomationRegistryLogicB2_3.sol

```diff
@@ -295,6 +298,32 @@ contract AutomationRegistryLogicB2_3 is AutomationRegistryBase2_3 {
     emit AdminPrivilegeConfigSet(admin, newPrivilegeConfig);
   }
 
+  /**
+   * @notice settles NOPs' LINK payment offchain
+   */
+  function settleNOPsOffchain() external {
+    _onlyFinanceAdminAllowed();
+    if (s_payoutMode == PayoutMode.ON_CHAIN) revert MustSettleOnchain();
+
+    uint256 length = s_transmittersList.length;
+    uint256[] memory balances = new uint256[](length);
+    for (uint256 i = 0; i < length; i++) {
+      address transmitterAddr = s_transmittersList[i];
+      uint96 balance = _updateTransmitterBalanceFromPool(transmitterAddr, s_hotVars.totalPremium, uint96(length));
```

Comment:
@RyanRHall  do we need to update the reserve account like 
`s_reserveAmounts[address(i_link)] = s_reserveAmounts[address(i_link)] - balance;`?
i assume no bc using offchain settlement means there is no LINK liquidity in service chain.